packages/pwa-module/view/frontend/layout/default.xml

```diff
@@ -1,13 +1,8 @@
 <?xml version='1.0'?>
-<page
-    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
-    xsi:noNamespaceSchemaLocation='urn:magento:framework:View/Layout/etc/page_configuration.xsd'>
+<page xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:noNamespaceSchemaLocation='urn:magento:framework:View/Layout/etc/page_configuration.xsd'>
     <body>
         <referenceContainer name='root'>
-            <block
-                class='Magento\Pwa\Block\Bundle'
-                name='Magento_Pwa_Block_Bundle'
-            />
+            <block class='Magento\Pwa\Block\Bundle' name='Magento_Pwa_Block_Bundle' />
         </referenceContainer>
     </body>
 </page>
```

Comment:
🤕 _sorry!_

Reply:
You're sorry about the PR or sorry about having a different opinion? :)